### ISSUE TITLE
Ensure history list and view is accessible

### ISSUE THREAD

**Issue by NightJar** [silverstripe/silverstripe-versioned-admin#20]
Probably a duplicate issue. But worth checking.

**Comment by robbieaverill** [comment]
@chillu noted we could make the list items into anchors (within a list) rather than table rows which would mean we'd get accessibility for free, alternatively we might be able to add `role` and `tabIndex` attributes to the table rows to achieve the same thing with less DOM change. Note that eslint enforces role and tabindex for elements such as spans that are acting as anchors

**Comment by chillu** [comment]
Thanks for summarising! To be clear, keyboard navigation is essential (and declaring clickable items for screen readers). Using links is optional, but would be preferrable since it allows native browser features like "copy link" or "open in new tab" to work. This should be the baseline for all CMS features we build.

**Comment by robbieaverill** [comment]
Fixed in https://github.com/creative-commoners/silverstripe-versioned-admin/pull/25 as part of our team's work on compare mode

**Comment by robbieaverill** [comment]
Actually not fixed in #33...

**Comment by robbieaverill** [comment]
#33 now fixes this
